tests/test_css.py

```diff
@@ -99,7 +99,8 @@ def test_function_content(self):
     def test_render(self):
         css = Css(self.css_dict)
         rendered_css = css.render()
-        expected_counter = Counter('<style> {} html { background-color: lightblue; } h1 { color: white; text-align: center; } </style>')
+        expected_counter = Counter('<style> {} html { background-color: lightblue; }'
+                                   'h1 { color: white; text-align: center; } </style>')
```

Comment:
This also breaks the unittest for it adds characters that make the assertion fail.